Emby.Server.Implementations/SocketSharp/WebSocketSharpRequest.cs

```diff
@@ -310,16 +310,9 @@ public static ReadOnlySpan<char> LeftPart(ReadOnlySpan<char> strVal, char needle
             return pos == -1 ? strVal : strVal.Slice(0, pos);
         }
 
-        private string pathInfo;
         public string PathInfo
         {
-            get
-            {
-                if (this.pathInfo is null)
-                    this.pathInfo = this.request.Path.Value;
-
-                return this.pathInfo;
-            }
+            get => this.request.Path.Value;
```

Comment:
Isn't this valid?: `public string PathInfo => request.Path.Value;`

Reply:
It is indeed: 

> Starting with C# 6, read-only properties can implement the get accessor as an expression-bodied member. In this case, neither the get accessor keyword nor the return keyword is used.